src/Swashbuckle.AspNetCore.Annotations/AnnotationsSchemaFilter.cs

```diff
@@ -96,6 +96,12 @@ private void ApplySchemaAttribute(OpenApiSchema schema, SwaggerSchemaAttribute s
 
             if (schemaAttribute.Title != null)
                 schema.Title = schemaAttribute.Title;
+
+            if (schemaAttribute.Example != null)
+                schema.Example = OpenApiAnyFactory.CreateFromJson(
+                    schemaAttribute.Example.All(char.IsDigit)
```

Comment:
Rather than second guessing based on the contents of the provided example (i.e `All(char.IsDigit)`), I think it's safer (and consistent with the implementation for XML comments) to wrap the example in quotes based on the schema type:

```csharp
var exampleAsJson = (schema.ResolveType(context.SchemaRepository) == "string") && !exampleNode.Value.Equals("null")
    ? $"\"{exampleNode.ToString()}\""
    : exampleNode.ToString();
```

NOTE: `context` is available on the calling method so can be passed down here.

Reply:
Other than that all looks good 👍

Reply:
Hi, @domaindrivendev 
Thanks for your answer.
In my tests the context.SchemaRepository is comming null, so I can't evaluate the type from the schema.
Am I doing something wrong?